src/shared/types.ts

```diff
@@ -60,9 +63,11 @@ export const ScheduledSurfaces: ScheduledSurface[] = [
     guid: 'NEW_TAB_DE_DE',
     ianaTimezone: 'Europe/Berlin',
     prospectTypes: [
-      ProspectType.GLOBAL,
-      ProspectType.ORGANIC_TIMESPENT,
+      ProspectType.COUNTS,
```

Comment:
could you provide lil bit context on why don't have recommend for berlin and US, but for Europe and Asia.

Reply:
will follow-up in doc.